### PR TITLE
Data transform update names

### PR DESCRIPTION
**Pull request by Tom-Szendrey**
# Description
The context for the user (aka d1 is mapped to [dataset name]) should not be within the first cell.
More context can be found here: https://github.com/DARPA-ASKEM/terarium/issues/5473

Screenshot of solution:

image

image

### CODE REVIEW

**Review by shawnyama — APPROVED** [review]

File: packages/client/hmi-client/src/components/dataset/tera-dataset-jupyter-panel.vue [hunk 1]
```diff
@@ -85,23 +85,28 @@
...
-			@new-dataset-saved="onNewDatasetSaved"
-			@download-response="onDownloadResponse"
-			@update-language="(lang) => emit('update-language', lang)"
-			@update-selected-outputs="(outputs) => emit('update-selected-outputs', outputs)"
-			:notebook-session="props.notebookSession"
-		/>
+		<div>
+			<div v-for="(data, indx) in dataContextDescription" :key="indx">
+				<input class="context-description" :value="data" disabled type="text" />
+			</div>
```
Comment (mwdchang) [inline packages/client/hmi-client/src/components/dataset/tera-dataset-jupyter-panel.vue:91]: Any particular reason why these are disabled `<input>` elements? Are we trying for some specific effect?

Comment (Tom-Szendrey) [reply packages/client/hmi-client/src/components/dataset/tera-dataset-jupyter-panel.vue:91]: no deeper meaning aside from `disabled` background colour and grab any rules we have on this that I may have missed

Comment (mwdchang) [reply packages/client/hmi-client/src/components/dataset/tera-dataset-jupyter-panel.vue:91]: Maybe an unordered-list is better at conveying what the code is trying to do. The background colour is like `var(--surface-100)` or something similar to that.

Comment (YohannParis) [reply packages/client/hmi-client/src/components/dataset/tera-dataset-jupyter-panel.vue:91]: Are we really shortening `index` with `indx` to save one character?

Comment (Tom-Szendrey) [reply packages/client/hmi-client/src/components/dataset/tera-dataset-jupyter-panel.vue:91]: No we are shortening it to idx to save two characters :P

**Review by mwdchang — APPROVED** [review]

**Pull request merged**